Quick Start guide is broken

### System Info

n/a

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### 🐛 Describe the bug

Attempting to run step 3 [here](https://llama-stack.readthedocs.io/en/latest/getting_started/index.html#quickstart) leads to the following error

### Error logs

```
$ uv run --with llama-stack-client,fire,requests demo_script.py
INFO:httpx:HTTP Request: GET http://localhost:8321/v1/models "HTTP/1.1 200 OK"
Traceback (most recent call last):
  File "/home/nathan/ai/llama-stack/playground/demo_script.py", line 9, in <module>
    model_id = next(m for m in models if m.model_type == "llm").identifier
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration
```

### Expected behavior

```
rag_tool> Ingesting document: https://www.paulgraham.com/greatwork.html

prompt> How do you do great work?

inference> [knowledge_search(query="What is the key to doing great work")]

tool_execution> Tool:knowledge_search Args:{'query': 'What is the key to doing great work'}

tool_execution> Tool:knowledge_search Response:[TextContentItem(text='knowledge_search tool found 5 chunks:\nBEGIN of knowledge_search tool results.\n', type='text'), TextContentItem(text="Result 1:\nDocument_id:docum\nContent:  work. Doing great work means doing something important\nso well that you expand people's ideas of what's possible. But\nthere's no threshold for importance. It's a matter of degree, and\noften hard to judge at the time anyway.\n", type='text'), TextContentItem(text="Result 2:\nDocument_id:docum\nContent:  work. Doing great work means doing something important\nso well that you expand people's ideas of what's possible. But\nthere's no threshold for importance. It's a matter of degree, and\noften hard to judge at the time anyway.\n", type='text'), TextContentItem(text="Result 3:\nDocument_id:docum\nContent:  work. Doing great work means doing something important\nso well that you expand people's ideas of what's possible. But\nthere's no threshold for importance. It's a matter of degree, and\noften hard to judge at the time anyway.\n", type='text'), TextContentItem(text="Result 4:\nDocument_id:docum\nContent:  work. Doing great work means doing something important\nso well that you expand people's ideas of what's possible. But\nthere's no threshold for importance. It's a matter of degree, and\noften hard to judge at the time anyway.\n", type='text'), TextContentItem(text="Result 5:\nDocument_id:docum\nContent:  work. Doing great work means doing something important\nso well that you expand people's ideas of what's possible. But\nthere's no threshold for importance. It's a matter of degree, and\noften hard to judge at the time anyway.\n", type='text'), TextContentItem(text='END of knowledge_search tool results.\n', type='text')]

inference> Based on the search results, it seems that doing great work means doing something important so well that you expand people's ideas of what's possible. However, there is no clear threshold for importance, and it can be difficult to judge at the time.

To further clarify, I would suggest that doing great work involves:

* Completing tasks with high quality and attention to detail
* Expanding on existing knowledge or ideas
* Making a positive impact on others through your work
* Striving for excellence and continuous improvement

Ultimately, great work is about making a meaningful contribution and leaving a lasting impression.
```

I attempted to start my LLS server with the following command instead:

```bash
ENABLE_OLLAMA=ollama OLLAMA_INFERENCE_MODEL="llama3.2:3b" llama stack build --template starter --image-type venv --run
```

This yielded different errors:

**Client**
```bash
Traceback (most recent call last):
  File "/home/nathan/ai/llama-stack/playground/demo_script.py", line 29, in <module>
    client.tool_runtime.rag_tool.insert(
  File "/home/nathan/.cache/uv/archive-v0/DMraTKr4vc5tew7FsXd0V/lib64/python3.12/site-packages/llama_stack_client/resources/tool_runtime/rag_tool.py", line 75, in insert
    return self._post(
           ^^^^^^^^^^^
  File "/home/nathan/.cache/uv/archive-v0/DMraTKr4vc5tew7FsXd0V/lib64/python3.12/site-packages/llama_stack_client/_base_client.py", line 1232, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nathan/.cache/uv/archive-v0/DMraTKr4vc5tew7FsXd0V/lib64/python3.12/site-packages/llama_stack_client/_base_client.py", line 1034, in request
    raise self._make_status_error_from_response(err.response) from None
llama_stack_client.InternalServerError: Error code: 500 - {'detail': 'Internal server error: An unexpected error occurred.'}
```

**Server**
```bash
ERROR    2025-07-16 10:53:38,463 __main__:249 server: Error executing endpoint route='/v1/tool-runtime/rag-tool/insert' method='post': There was a    
         specific connection error when trying to load sentence-transformers/all-MiniLM-L6-v2:                                                        
         401 Client Error: Unauthorized for url: https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/config.json (Request ID:  
         Root=1-6877bcf2-640bdc8871da2cd13931ec4c;72b0ccdd-28a5-447d-abb6-a8cd8b4b0a21)                                                               
                                                                                                                                                      
         Invalid credentials in Authorization header                                                                                                  
INFO:     ::1:49188 - "POST /v1/tool-runtime/rag-tool/insert HTTP/1.1" 500 Internal Server Error
14:53:38.555 [END] /v1/tool-runtime/rag-tool/insert [StatusCode.OK] (349.41ms)
 14:53:38.464 [ERROR] Error executing endpoint route='/v1/tool-runtime/rag-tool/insert' method='post': There was a specific connection error when trying to load sentence-transformers/all-MiniLM-L6-v2:
401 Client Error: Unauthorized for url: https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/config.json (Request ID: Root=1-6877bcf2-640bdc8871da2cd13931ec4c;72b0ccdd-28a5-447d-abb6-a8cd8b4b0a21)

Invalid credentials in Authorization header
```

--- a/docs/source/getting_started/quickstart.md
+++ b/docs/source/getting_started/quickstart.md
@@ -19,7 +19,7 @@ ollama run llama3.2:3b --keepalive 60m
 #### Step 2: Run the Llama Stack server
 We will use `uv` to run the Llama Stack server.
 ```bash
-INFERENCE_MODEL=llama3.2:3b uv run --with llama-stack llama stack build --template starter --image-type venv --run
+ENABLE_OLLAMA=ollama OLLAMA_INFERENCE_MODEL=llama3.2:3b uv run --with llama-stack llama stack build --template starter --image-type venv --run
 ```
 #### Step 3: Run the demo
 Now open up a new terminal and copy the following script into a file named `demo_script.py`.
@@ -110,6 +110,12 @@ To further clarify, I would suggest that doing great work involves:
 Ultimately, great work is about making a meaningful contribution and leaving a lasting impression.
 ```
 Congratulations! You've successfully built your first RAG application using Llama Stack! 🎉🥳
+
+```{admonition} HuggingFace access
+:class: tip
+
+If you are getting a **401 Client Error** from HuggingFace for the **all-MiniLM-L6-v2** model, try setting **HF_TOKEN** to a valid HuggingFace token in your environment
+```
 
 ### Next Steps
 

